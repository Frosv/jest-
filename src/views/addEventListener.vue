<template>
  <div>axios</div>
</template>

<script>
export default {
  mounted () {
    this.init()
  },
  methods: {
    init () {
      window.addEventListener('test',() =>{
        console.log(123)
      })
    }
  }
}
</script>
