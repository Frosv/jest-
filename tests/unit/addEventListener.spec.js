import {
  createLocalVue,
  shallowMount
} from '@vue/test-utils'
import addEventListener from '@/views/addEventListener.vue'

describe('addEventListener', () => {
  global.addEventListener = jest.fn();
  const wrapper = shallowMount(addEventListener)

  it('1', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
