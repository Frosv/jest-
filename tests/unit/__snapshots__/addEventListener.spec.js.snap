// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addEventListener 1 1`] = `<div>axios</div>`;
